src/PrimeFuncPack.Core/core_objects/Objects.Tests/Objects.Tests.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <IsPackable>false</IsPackable>
+    <Description>PrimeFuncPack: A Functional Programming Pack for .NET</Description>
+    <Copyright>Copyright © 2020 Andrei Sergeev, Pavel Moskovoy</Copyright>
+    <RootNamespace>PrimeFuncPack.DependencyPipeline</RootNamespace>
```

Comment:
Seems two different RootNamespace tags in the same csproj

Reply:
Fixed